### PR TITLE
Patch review

### PR DESCRIPTION
**Pull request by Susizhou**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -215,6 +215,54 @@ describe("/api/reviews/:review_id", () => {
       });
     });
 
+    test('works ith decreasing number of votes', () => {
```
Comment (MKD01) [inline __tests__/app.test.js]: great test but missing a w on the second word there.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -215,6 +215,54 @@ describe("/api/reviews/:review_id", () => {
...
+              'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+            review_body: 'Farmyard fun!',
+            category: 'euro game',
+            created_at: expect.any(String),
+            votes: 0
+        })
+      });
+    });
+
+    test('works ith decreasing number of votes into negative voting', () => {
```
Comment (MKD01) [inline __tests__/app.test.js]: also like this test but this also has a missing w on the second word

File: __tests__/app.test.js [hunk 3]
```diff
@@ -123,7 +123,99 @@ describe("/api/reviews/:review_id", () => {
...
+
+    test('give error if invalid id is given', () => {
+      const updateInfo = {
+        inc_votes: 1
+      }
+
+      return request(app)
+        .patch("/api/reviews/100")
+        .send(updateInfo)
+        .expect(404)
```
Comment (MKD01) [inline __tests__/app.test.js:172]: the description to this test implies that 100 is invalid, but this is in fact a valid ID it simply does not exist.

File: __tests__/app.test.js [hunk 4]
```diff
@@ -123,7 +123,99 @@ describe("/api/reviews/:review_id", () => {
...
+        .then(({ body }) => {
```
Comment (MKD01) [inline __tests__/app.test.js:173]: This test description implies 100 is an invalid ID, but this is in fact a valid ID it simply just does not exist.

File: models/games.models.js [hunk 5]
```diff
@@ -60,17 +60,42 @@ exports.addComment = (review_id, body_req) => {
...
+  const { inc_votes } = body;
+
+  if (!inc_votes) {
+    return Promise.reject({
+      status: 400,
+      msg: "Input data format was not correct",
+    });
+  }
+  return this.fetchReviewsByID(review_id).then(()=>{
+    return db.query(
```
Comment (MKD01) [inline models/games.models.js:89]: checking review in this way is unnecessary here. I would suggest taking this out and seeing what you get back from this query if given an invalid review_id, and then checking what you get back from this and how that can be used to Promise.reject with a status code and error message.

**Review by MKD01 — APPROVED** [review]
Please do make those changes first but once thats done im happy for you to merge 👍

File: models/games.models.js (hunk 5 above)
Comment (Susizhou) [reply models/games.models.js:89]: I have tried to remove the calling of the other function, but it does not give me any error from SQL, it gives me a 201. I did get an error from SQL in the case of the post. but i didnt for this one.

**Pull request merged**